styles/globals.css

```diff
@@ -15,24 +15,36 @@
 
   body,
   #__next {
-    @apply flex min-h-full flex-col text-base leading-relaxed;
+    @apply flex min-h-full flex-col text-base text-uog-color-body-copy leading-relaxed;
     font-size: 1.8rem;
   }
 
   #__next {
     @apply flex-1;
   }
 
-  h1,
-  h2,
-  h3,
-  h4,
-  h5,
-  h6 {
+  h1, h3, h4, h5, h6 {
+    @apply text-uog-color-black;
+  }
+
+  h2 {
+    @apply text-uog-color-red;
```

Comment:
Homepage ends up with a red colour

<img width="924" alt="image" src="https://github.com/user-attachments/assets/c4a1b384-88ae-40b4-8854-79bb307263ba" />

/ovc/news - I'm seeing a red colour as well in the custom footer
<img width="482" alt="image" src="https://github.com/user-attachments/assets/307e59de-e205-41e2-b583-622d423e6d79" />

Reply:
Fixed by adding the appropriate colour classes we configured earlier (which is the proper way to do it - as I understand things, the default colour for `h2` elements is supposed to be the Brand Guide red and if people wish to override that in a component, they need to specify an alternate colour class).